react-accordion-component/src/accordian.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Accordian({ items }) {
+  const [isOpen, setIsOpen] = useState('');
+
+  return (
+    <div className='row'>
+      <ListItems items={items} isOpen={isOpen} setIsOpen={setIsOpen} />
+    </div>
+  );
+}
+
+function randomColor() {
+  const classes = {
+    width: '25%',
+    backgroundColor: 'rgb(' + Math.floor(Math.random() * 255) +
+      ',' + Math.floor(Math.random() * 255) + ',' +
+      Math.floor(Math.random() * 255) + ')'
+  };
+  return classes;
+}
+
+function ListItems({ items, isOpen, onClick, setIsOpen }) {
+  return (
+    <>
+      {items.map((item) => (
```

Comment:
It would make more sense from a component standpoint to have a component that displays just one language. It's nice to keep the components small and single-focused. That way they are more reusable.

So I would suggest moving the `items.map` code up into Accordian, or break this into two components.